### PR TITLE
Freeze revision for country-codes

### PR DESCRIPTION
**Pull request by shawncruz**
Slack thread: https://flowio.slack.com/archives/C0B6M38K0/p1656093759090969

While attempting to add a new payment method to our references, I encountered the following error while running the `release.go` script:
```shell
> go run reference.go all
Downloading data...
------------------------------
Downloading https://raw.githubusercontent.com/bdswiss/country-language/master/data.json...
  -> Stored in data/source/languages.json
Downloading https://raw.githubusercontent.com/datasets/country-codes/master/data/country-codes.csv...
  -> Stored in data/source/countries.csv
Downloading http://dev.maxmind.com/static/csv/codes/country_continent.csv...
  -> Stored in data/source/country-continents.csv
Downloading https://raw.githubusercontent.com/unicode-cldr/cldr-numbers-full/master/main/en-US-POSIX/currencies.json...
  -> Stored in data/source/cldr-currencies.json

Cleansing data...
------------------------------
Writing data/cleansed/languages.json
Writing data/cleansed/locale-names.json
Writing data/cleansed/countries.json
Writing data/cleansed/numbers.json
Writing data/cleansed/currency-symbols.json
Writing data/cleansed/currencies.json
Writing data/cleansed/country-duties.json
Writing data/cleansed/carriers.json
Writing data/cleansed/carrier-services.json
Writing data/cleansed/provinces.json
Writing data/cleansed/province-translations.json
Writing data/cleansed/country-continents.json
Writing data/cleansed/payment-methods.json
Writing data/cleansed/timezones.json
Writing data/cleansed/country-timezones.json
Writing data/cleansed/country-default-languages.json
Writing data/cleansed/currency-locales.json

Generating final models...
------------------------------
Invalid currency code: VES
exit status 1
```

When running all scripts, there’s a mismatch between currencies [here](https://github.com/flowcommerce/json-reference/blob/main/final/final.go#L692-L696), because `data/source/countries.csv` defines VES as the official currency of Venezuela, while we still define it as VEF [here](https://github.com/flowcommerce/json-reference/blob/main/data/original/currencies.json#L662-L666).

 `data/source/countries.csv` comes from [this](https://raw.githubusercontent.com/datasets/country-codes/master/data/country-codes.csv) external source. This source is not currently pinned to a version, so any changes to this source will be introduced into our system. Since we explicitly reverted our "support" for VES [here](https://github.com/flowcommerce/json-reference/pull/103), we're no longer in sync with the datasets source. To resolve this, my proposed solution is to pin the version of country-codes that we use that directly precedes the changes in which VES was added as the default currency of Venezuela:
- Commit where VES is updated to default currency of Venezuela: https://github.com/datasets/country-codes/commit/060e8c02fc8d9281a294669b89a0e50b5164abd4
- Commit directly preceding change: https://github.com/datasets/country-codes/commit/2ed03b6993e817845c504ce9626d519376c8acaa

My proposed solution here is to pin the version country-codes to the commit directly preceding  https://github.com/datasets/country-codes/commit/060e8c02fc8d9281a294669b89a0e50b5164abd4.

### CODE REVIEW

**Review by mbryzek — APPROVED** [review]
+1 to pegging the version of countries.

To know that this is going to be fine, would be great to take the ISO codes that change here (countries, currencies) and verify in a few services (experience, catalog and metric are likely good candidates) that these values are not used. Otherwise we would see a runtime error in production from the use of Countries.mustFind and Currencies.mustFind

**Comment by shawncruz** [comment]
@mbryzek That suggestion makes sense; will do 👍🏻 . 

It might be more effort than it's worth, but another idea was to fiddle around with the revisions on our countries and currencies sources until we don't see a diff.

**Comment by mbryzek** [comment]
> It might be more effort than it's worth

Long term, we have previously discussed just keeping our own static list of reference data to simplify all of this

File: data/cleansed/countries.json [hunk 1]
```diff
@@ -832,13 +832,6 @@
     "iso_3166_3": "KIR",
     "currency": "AUD"
   },
-  {
-    "name": "Kosovo",
-    "continent": "AS",
-    "iso_3166_2": "XK",
-    "iso_3166_3": "RKS",
-    "currency": "EUR"
-  },
```
Comment (benwaffle) [inline data/cleansed/countries.json:841]: this looks like a revert of https://github.com/flowcommerce/json-reference/commit/9c50fd4584a05b4a64f6f5e2973a838e53ee0585, i think we need to keep it for backwards-compatibility

File: data/cleansed/countries.json [hunk 2]
```diff
@@ -1546,6 +1532,13 @@
     "iso_3166_3": "THA",
     "currency": "THB"
   },
+  {
+    "name": "The former Yugoslav Republic of Macedonia",
```
Comment (benwaffle) [inline data/cleansed/countries.json:1536]: This looks like a revert of https://github.com/flowcommerce/json-reference/commit/c0c144a7da24be65a833779e765cd01a7be1283c, we should undo this change

**Comment by shawncruz** [comment]
@benwaffle Good callouts. Were these changes manually made or could I revert our data sources to a specific commit that would undo these changes? Reason I ask is because next time someone has to run this script, they'll have to deal with this problem as well.

**Comment by benwaffle** [comment]
I think they were manually made by changing `data/source/countries.csv` and skipping the `download` step

**Comment by shawncruz** [comment]
No longer blocked by this, so closing out for now. Resolution to my blocked work is outlined here: https://github.com/flowcommerce/json-reference/pull/115#issuecomment-1165791872
